packages/proxy/package.json

```diff
@@ -46,12 +46,12 @@
     "@verdaccio/streams": "workspace:11.0.0-alpha.3",
     "@verdaccio/utils": "workspace:6.0.0-6-next.5",
     "JSONStream": "1.3.5",
-    "lodash": "4.17.20",
+    "lodash": "4.17.21",
     "request": "2.87.0"
   },
   "devDependencies": {
     "@verdaccio/types": "workspace:11.0.0-6-next.7",
-    "nock": "13.0.11",
+    "nock": "12.0.3",
```

Comment:
why the downgrade here?

Reply:
Ok, because there was a major version difference 12/13 I felt it safer to do for the lowest common denominator. Nevertheless if 13.0.11 works in build & test you are right to prefer upgrade instead on downgrade.
In fact lets try latest v13.1.3.

Reply:
Hmm, unfortunately when syncing all nock deps to 13.0.11 some tests fail and I have no idea while.
If nock deps are aligned to v12.0.3 all tests pass.
Not sure what to do. Really want to have same version of nock in all packages.

Reply:
Yeah I remember `13.0.11` needs some upgrades, that's that explains why not all packages are aligned, we don't need all packages aligned at this point.  Let's keep the versions that does not break the test and we are good. `nock` will be removed eventually since I'm moving to `undici` instead `node-fetch` or `request` and `nock` does not support `undici`.